inference_server/launcher/service/launcher.py

```diff
@@ -0,0 +1,57 @@
+"""
+Account Service
+
+This microservice handles the lifecycle of Accounts
```

Comment:
This looks like it is copy-pasted from an example, needs to be adjusted for the actual use at hand.